bot/src/main/java/org/openjdk/skara/bot/BotRunner.java

```diff
@@ -237,6 +237,10 @@ boolean isReady() {
         return isReady;
     }
 
+    boolean isHealthy() {
+        return true;
```

Comment:
I think this implementation is a tad optimistic regarding the health of the runner. :) But I also think defining healthy can be tricky.

Reply:
I don't have any better idea right now. We may want to revisit this later.